docs/reference/ml/anomaly-detection/apis/get-job-stats.asciidoc

```diff
@@ -158,24 +159,28 @@ NOTE: Unless there is at least one forecast, `memory_bytes`, `records`,
 --
 
 `forecasts_stats`.`forecasted_jobs`:::
-(long) The number of jobs that have at least one forecast.
+(long) A value of `0` indicates that forecasts do not exist for this job. A 
+value of `1` indicates that at least one forecast exists.
 
 `forecasts_stats`.`memory_bytes`:::
-(object) Statistics about the memory usage: minimum, maximum, average and total.
+(object) The `avg`, `min`, `max` and `total` memory usage in bytes for forecasts 
+related to this job. Omitted if forecasts do not exist.
```

Comment:
I generally like starting the sentence with the conditional part, so folks can stop reading if it doesn't apply to them.
```suggestion
related to this job. If there are no forecasts, this property is omitted.
```